packages/flutter/lib/src/painting/_network_image_io.dart

```diff
@@ -15,13 +15,15 @@ import 'image_provider.dart' as image_provider;
 import 'image_stream.dart';
 
 /// The dart:io implemenation of [image_provider.NetworkImage].
-class NetworkImage extends image_provider.ImageProvider<image_provider.NetworkImage> implements image_provider.NetworkImage {
+class NetworkImage
```

Comment:
Please revert all of the formatting related changes.

The Flutter repository does not use dartfmt - see our style guide for more information (https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo)

Reply:
Fixed. Check it again?